eng/convert-codecoveragetoxml.ps1

```diff
@@ -0,0 +1,18 @@
+[cmdletbinding()]
+param(
+    [Parameter(Mandatory=$True)]
+    [string]$Path
+)
+
+Set-StrictMode -Version 2.0
+
+$coverageFile = Get-ChildItem -Path $Path -Filter *.coverage -Recurse -ErrorAction SilentlyContinue -Force
+Write-Host "Code coverage files found: "
+Write-Host $coverageFile.FullName
+
+$codeCoverageExe = "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Team Tools\Dynamic Code Coverage Tools\CodeCoverage.exe"
```

Comment:
Wondering if these code coverage tools are available with the VS build tools, or is this PR adding a dependency on full VS being installed in the pools?

If it's the second, is that acceptable? I know we've been trying to get product repos to only depend on full visual studio installs for UI testing that isn't possible with just the build tools.

Reply:
I have an issue open to find this value programmatically, and will ensure that this is installed with the BuildTools version of VS. If not, we'll have to install it: https://github.com/dotnet/core-eng/issues/8466

Reply:
@markwilkie and @Chrisboh, making sure you guys are okay with me refactoring the dependency on full-version of Visual Studio in another issue. Currently, this is how we're doing it for Helix, as well, but it's something I want to make programmatic, or download and install it if it can't be found. (So, if we can't find it on a build machine that has BuildTools, it would download and install).

Reply:
So long at we're not regressing, I'm cool.  Also, would really love to lose the full VS dependency...